apps/src/storage/dataBrowser/ColumnHeader.jsx

```diff
@@ -48,6 +48,7 @@ class ColumnHeader extends React.Component {
     isEditable: PropTypes.bool.isRequired,
     isEditing: PropTypes.bool.isRequired,
     isPending: PropTypes.bool.isRequired,
+    readOnly: PropTypes.bool,
```

Comment:
It looks like the readOnly prop is passed around between multiple components. Is this something that could be moved to redux?

Reply:
It's only passed from Parent to Child (DataTable -> ColumnHeader and DataTable -> EditTableRow), and my understanding was that the best practice for passing data from a parent to a child is to use props not state. What's the benefit of moving to redux state?

Reply:
I see the benefit is that pulling from redux better guarantees that these components have synced state. However, since there is a tree structure for these props, I can think it's fine to leave as is.